osde2e/README.md

```diff
@@ -98,6 +98,21 @@ A new machinepool can be created on each cluster after installation if `--machin
 | --machinepool-wait | Wait until number of ready nodes equals number of required replicas | False |
 | --machinepool-wait-cycles | Number of 5 seconds wait cycles until halts the waiting | 60 |
 
+### Wildcard variable
```

Comment:
Can we add a ```--wildcard-options``` line in the optional args section for osde2e and rosa that points to their respective wildcard variable doc sections? If a user is just scanning the variables available they would never see it as it's written today.

Reply:
good idea, done